src/index.js

```diff
@@ -0,0 +1,196 @@
+                    <li><a href="#" class="text-white">Email me</a></li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="navbar navbar-dark bg-dark shadow-sm">
+            <div class="container">
+              <a href="#" class="navbar-brand d-flex align-items-center">
+                <i class="fas fa-users"></i>
+                <strong>${teamName.teamName}</strong>
+              </a>
+            </div>
+          </div>
+        </header>
+    
+        <main>
+          <section class="py-5 text-center container">
+            <div class="row py-lg-5">
+              <div class="mx-auto">
+                <h1 class="fw-light">🎉 Welcome to ${teamName.teamName}! 🎉</h1>
+                <p class="lead text-muted">
+                  Our team members are below, if you like the look of us and you
+                  think we can work together, why not fire over an email? 🚀
+                </p>
+              </div>
+            </div>
+          </section>
+  
+            <div class="album py-5 bg-light">
+            <div class="container">
+              <div class="d-flex flex-row flex-wrap">${cardSection}</div>
+            </div>
+          </div>
+        </main>
+    
+        <footer class="text-muted py-5 bg-dark">
+          <div class="container text-center text-white">
+            <p class="mb-1">
+              Team Profile Generator, made by
+              <a href="https://github.com/conorjkelly96">Conor Kelly</a>.
+            </p>
+          </div>
+        </footer>
+    
+        <script src="../assets/dist/js/bootstrap.bundle.min.js"></script>
+      </body>
+    </html>
+    `;
+};
+
+// initialize user interaction
+const init = async () => {
+  // prompt Main Questions
+  let inProgress = true;
+  const teamMembers = [];
+
+  const answers = await inquirer.prompt(teamBuilderQuestions);
+  const { name, id, email, officeNumber } = await inquirer.prompt(
+    managerQuestion
+  );
+  const manager = new Manager({ name, id, email, officeNumber });
+  teamMembers.push(manager);
+
+  while (inProgress) {
+    const { employeeType } = await inquirer.prompt(employeeTypeQuestion);
+
+    if (employeeType == "Engineer") {
+      const { name, id, email, github } = await inquirer.prompt(
+        engineerQuestions
+      );
+      const engineer = new Engineer({ name, id, email, github });
+      teamMembers.push(engineer);
+    }
+
+    if (employeeType == "Intern") {
+      const { name, id, email, school } = await inquirer.prompt(
+        internQuestions
+      );
+      const intern = new Intern({ name, id, email, school });
+      teamMembers.push(intern);
+    }
+
+    const anotherEmployee = await inquirer.prompt(continueQuestion);
+    if (!anotherEmployee.newMember) {
+      inProgress = false;
+    }
+  }
+
+  // generate HTML file
+  const htmlPage = generateHTMLfile(answers, teamMembers);
```

Comment:
Pass `teamName` from the destructuring on line 155, rather than the whole object.

Reply:
resolved